src/main/java/com/sparta/popupstore/s3/S3ImageController.java

```diff
@@ -32,4 +35,13 @@ public ResponseEntity<S3UrlResponseDto> getPromotionEventImagePreSignedUrl(
                 .status(HttpStatus.CREATED)
                 .body(imageService.getPreSignedUrl("promotionevent", imageRequestDto.getFileName()));
     }
+
+    @GetMapping("/popup-stores/image/preassigned")
+    public ResponseEntity<List<S3UrlResponseDto>> getPopupStoreImagePreSignedUrl(
+            @RequestBody @NotEmpty(message = "이미지를 하나이상 올려주세요") List<ImageRequestDto> imageRequestDto
```

Comment:
```suggestion
            @RequestBody @NotEmpty(message = "이미지를 하나이상 올려주세요") List<ImageRequestDto> imageRequestList
```
변수 이름으로 list임을 알 수 있도록 해주세요.